src/makeRobot.js

```diff
@@ -39,6 +39,69 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robotInfo = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      // That bcs max line in Lint == 80
+      const firstPart = `name: ${this.name}, chip version: `;
+      const secondPart = `${this.version}, wheels: ${this.wheels}`;
+
+      return firstPart + secondPart;
```

Comment:
Remove these variables and use concatenation string

```suggestion
     return `name: ${}`
     + ` chip version${}`;
```